http/client.go

```diff
@@ -88,9 +159,18 @@ func (c *client) Send(req cmds.Request) (cmds.Response, error) {
 	} else {
 		httpReq.Header.Set(contentTypeHeader, applicationOctetStream)
 	}
-	httpReq.Header.Set(uaHeader, config.ApiVersion)
```

Comment:
Are we doing this somewhere else?

Reply:
Yeah, we now pass in the userAgent when constructing the client, and then set it roughly around here

Reply:
Got it.